optionals/compat_sog/CfgAmmo/explosives.hpp

```diff
@@ -0,0 +1,63 @@
+class DirectionalBombBase;
+class vn_mine_m18_ammo: DirectionalBombBase {
+    EGVAR(Explosives,defuseObjectPosition)[] = {0, 0, 0.15};
```

Comment:
I think `Explosives` should be all lowercase.

Reply:
Fixed, also fixed an uppercase `EGVAR(minedetector,detectable)`. Didn't find any other occurances with regex.